### ISSUE TITLE
Using Tags to identify all the resources

### ISSUE THREAD

**Issue by dashbitla** [alienfast/elastic-beanstalk#38]
When I did the deployment into production environment, the EC2 instances created with production-a-xdfdsfdfs names and so are the other AWS resources with the similar names.

Is there a way to pass tags attributes to the EB deployment environment??

**Comment by rosskevin** [comment]
Honestly I've never heard of tags.  If you can point me to one of the AWS docs, I can tell you if it integrates.

**Comment by dashbitla** [comment]
Refer to:

https://github.com/ThoughtWorksStudios/eb_deployer/search?utf8=%E2%9C%93&q=tags

```
      def create_environment(app_name, env_name, stack_name, cname_prefix, version, tier, tags, settings)
        request = reject_nil({
          :application_name => app_name,
          :tier => environment_tier(tier),
          :cname_prefix => name_prefix,

**          :tags => tags**
        })
        @client.create_environment(request)
```

Refer to:
https://github.com/ThoughtWorksStudios/eb_deployer/blob/master/lib/eb_deployer.rb

```
    environment = Environment.new(application, env_name, stack_name, bs) do |env|
      env.resource_stacks = resource_stacks
      env.settings = opts[:option_settings] || opts[:settings] || []
      env.inactive_settings = opts[:inactive_settings] || []
      env.creation_opts = {
        :solution_stack => opts[:solution_stack_name],
        :cname_prefix =>  opts[:cname_prefix],
        :smoke_test => opts[:smoke_test],
        :phoenix_mode => opts[:phoenix_mode],
        :blue_green_terminate_inactive => opts[:blue_green_terminate_inactive] || false,
        :blue_green_terminate_inactive_wait => opts[:blue_green_terminate_inactive_wait] || 600,
        :blue_green_terminate_inactive_sleep => opts[:blue_green_terminate_inactive_sleep] || 15,
**        :tags => opts[:tags],**  # this gem is not passing the tags 
        :tier => opts[:tier]
      }
```

See here:

https://github.com/alienfast/elastic-beanstalk/blob/master/lib/elastic/beanstalk/tasks/eb.rake#L328

```
options = {
        application: EbConfig.app,
        environment: EbConfig.environment,
        version_label: find_option_app_version,
        solution_stack_name: EbConfig.solution_stack_name,
        option_settings: EbConfig.option_settings,
        inactive_settings: EbConfig.inactive_settings,
        strategy: EbConfig.strategy.to_sym,
        package: package
    }

    options[:package_bucket] = EbConfig.package_bucket unless EbConfig.package_bucket.nil?
    options[:keep_latest] = EbConfig.keep_latest unless EbConfig.keep_latest.nil?
    options[:version_prefix] = EbConfig.version_prefix unless EbConfig.version_prefix.nil?
    options[:tier] = EbConfig.tier unless EbConfig.tier.nil?

```

If we can do the following line added above:

`options[:tags] = EbConfig.tags unless EbConfig.tags.nil?`

supports:

```
package_bucket: 'app-name-124-deploy'
**tags: 
  Name: app-name124**

```
This will help name all the EC2 instances and other resources with name starting from **app-name124**-a-sdfsdfsd.

That will help identify all the resources ... especially useful when we have multiple apps / environments running under the same AWS login.

Hope the above is clear!

**Comment by rosskevin** [comment]
Feel free to send a PR with any of the additional options for eb_deployer, there isn't a reason I can see that we shouldn't support passing through those options.  We've just been adding them as needed.

**Comment by rosskevin** [comment]
Closed with #49, thanks @canatella
